DeepLearningSuite/build/CMakeFiles/3.5.1/CMakeCCompiler.cmake

```diff
@@ -0,0 +1,67 @@
+set(CMAKE_C_COMPILER "/usr/bin/cc")
```

Comment:
These type of files shouldn't be pushed

Reply:
@vinay0410  removed the build files

Reply:
will be removing the documentation commit as well , just a few more minutes

Reply:
@vinay0410  finished doing as required:
1.) Removed the build files (Sorry did not get at first and took so long)
2.)This PR shifts all logs to glog
3.) No other files are touched
Thank you very much explaining it so patiently , now have got better picture of git because of you